Source/SwiftLintBuiltInRules/Rules/Idiomatic/DuplicateImportsRule.swift

```diff
@@ -153,7 +153,7 @@ private extension DuplicateImportsRule {
 
         override func visit(_ node: CodeBlockItemListSyntax) -> CodeBlockItemListSyntax {
             let itemsToRemove = node
-                .enumerated()
+                .enumerated() // swiftlint:disable:next unused_enumerated
```

Comment:
I still wonder why this needs to be skipped. I think that the rule should either stop immediately when encountering a `filter` method or it should skip it and continue checking the subsequent `map`-like methods. If there aren't any of the latter, the rule remains silent.

Reply:
Ah - my misunderstanding - I would love to continue checking, but I couldn't find an elegant way to do that - I'll look again.

Or we can just stop on filter and ignore (as opposed to stopping after filter, which is what we're actually doing in the current code here).